README.md

```diff
@@ -1,2 +1,71 @@
-# Awesome-Books
-First Project in Module 2
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> This is our first pair-programming project in Module 2.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo
+
+Visit ...... to view online
```

Comment:
- [ ] [OPTIONAL] I kindly suggestt that you add the link to live demo if you have one or remove this section entirely